task_hive/.github/workflows/sonarcloud.yml

```
@@ -2,9 +2,9 @@ name: SonarQube Analysis
 
 on:
   push:
-    branches: [main]
+    branches: [ "main" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)